R/DualSimplexSolver.R

```diff
@@ -746,6 +756,40 @@ DualSimplexSolver <- R6Class(
     get_marker_genes = function() {
       private$finalize_first()
       return(self$st$marker_genes)
+    },
+
+    #' @description
+    #' Calculate V_row on the fly.
+    #' TODO: It might be good to have a more efficient implementation, if this function will be frequently called.
+    get_V_row = function(scaling) {
+      private$set_data_first()
+
+      dr <- matrixStats::rowProds(self$st$scaling$D_vs_row)
```

Comment:
Are you sure this will not cause type overflow if we multiply a lot of similar values to each other? As the result values become smaller and smaller and can be out of time values.  Iterative multiplication gave us at least some hope to stay in the same value range